.github/CONTRIBUTING.md

```diff
@@ -43,20 +43,22 @@ LintMD-2    1.54s ± 1%   1.54s ± 1%    ~     (p=0.604 n=10+10)
 
 To run the tests, you'll want to invoke either `make bench` or `make ci` (see [Setting up a Development Environment]() for more information).
 
-## <a name="devenv"></a> Setting up a Development Environment
+## Setting up a Development Environment
 
-You'll need to have [Ruby](https://www.ruby-lang.org/en/downloads/) (v2.3+) and [Go](https://golang.org/) (v1.7+) installed. Next, you'll need to install and configure [`aruba`](https://github.com/cucumber/aruba) (assuming you're inside the `vale` directory):
+Prerequisites:
+
+* [Ruby](https://www.ruby-lang.org/en/downloads/) (v2.3+)
+* [Go](https://golang.org/) (v1.7+) installed.
+* [Asciidoctor](http://asciidoctor.org/) available on your `$PATH`.
+* [rst2html](http://docutils.sourceforge.net/docs/user/tools.html#rst2html-py) available on your `$PATH`. The latter is installed with both [Sphinx](http://www.sphinx-doc.org/en/stable/) and [docutils](https://pypi.python.org/pypi/docutils).
 
 ```bash
-$ gem install bundler # if necessary
-$ aruba init --test-framework cucumber
+$ gem install bundler:2.2.31 # if necessary
```

Comment:
Added the specific version here, otherwise the install complained about me using a more recent recent version of bundler than the one used to generate the lockfile. I could have regenerated the lockfile but without 100% passing tests that seemed unsafe.